.github/NativeBuildReport.kts

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("jcenter","https://jcenter.bintray.com/")
+@file:MavenRepository("maven-central","https://repo1.maven.org/maven2/")
```

Comment:
I think we should use `https://repo.maven.apache.org/maven2/` instead.

Reply:
I actually never used that one, thanks!